src/patternfly/layouts/Grid/examples/grid-base-example.hbs

```diff
@@ -1,35 +1,35 @@
 {{#> grid}}
-  {{#> grid-item modifierClass="pf-m-12-col"}}
+  {{#> grid-item grid-item--modifiers="pf-m-12-col"}}
```

Comment:
This should be grid-item--modifier

Reply:
Changed this to plural form to match `grid-item--attributes` as multiple modifiers can be passed to partials.

Reply:
ah gotcha, makes sense.